### PR TITLE
smp svr sample configuration for nrf54l15

### PR DESCRIPTION
**Pull request by michalek-no**

### CODE REVIEW

**Review by nordicjm — COMMENTED** [review]
Needs adding to confluence page `Technical Debt`

File: samples/subsys/mgmt/mcumgr/smp_svr/sample.yaml [hunk 1]
```diff
@@ -12,9 +12,11 @@ tests:
       - nrf52840dk_nrf52840
       - pinnacle_100_dvk
       - mg100
+      - nrf54l15pdk_nrf54l15
```
Comment (gchwier) [inline samples/subsys/mgmt/mcumgr/smp_svr/sample.yaml:15]: nrf54l15pdk_nrf54l15_cpuapp?

**Review by gchwier — APPROVED** [review]

**Comment by michalek-no** [comment]
> Needs adding to confluence page `Technical Debt`

what do you mean by that?

**Comment by michalek-no** [comment]
rebase

File: dts/arm/nordic/nrf54l_common.dtsi [hunk 2]
```diff
@@ -18,7 +18,7 @@
 			rram0: rram@0 {
 				compatible = "soc-nv-flash";
 				erase-block-size = <4096>;
-				write-block-size = <1>;
+				write-block-size = <16>;
```
Comment (nordicjm) [inline dts/arm/nordic/nrf54l_common.dtsi:21]: Replace this PR description, fromlist -> fromtree, add the commit it is from and remove the upsteeam PR link

**Review by nordicjm — APPROVED** [review]

**Comment by michalek-no** [comment]
rebase

**Review by nvlsianpu — APPROVED** [review]

**Review by de-nordic — APPROVED** [review]

**Pull request merged**